src/PKSim.Core/Model/ParameterExtensions.cs

```diff
@@ -26,23 +27,43 @@ public static void SetPercentile(this IParameter parameter, double percentile)
          distributedParameter.Percentile = percentile;
       }
 
+      public static bool IsGlobalExpression(this ExpressionParameterUpdate parameter)
+      {
+         return parameter != null && isGlobalExpression(parameter);
+      }
+      
       public static bool IsGlobalExpression(this IParameter parameter)
       {
-         if (parameter == null)
-            return false;
+         return parameter != null && isGlobalExpression(parameter);
+      }
 
-         return AllGlobalRelExpParameters.Contains(parameter.Name);
+      private static bool isGlobalExpression(IWithName withName)
+      {
+         return AllGlobalRelExpParameters.Contains(withName.Name);
       }
 
       public static bool IsIndividualMoleculeGlobal(this IParameter parameter) =>
          CoreConstants.Parameters.AllGlobalMoleculeParameters.Contains(parameter.Name);
 
+      private static bool isExpression(IWithName parameter)
+      {
+         return isGlobalExpression(parameter) || parameter.IsNamed(REL_EXP);
+      }
+
+      public static bool IsExpression(this ExpressionParameterUpdate parameter)
```

Comment:
what's this class?`ExpressionParameterUpdate`
Having a hard time to understand what it is. Somekind of parameter?
Also why do we need extension methods on **concrete** types?

Reply:
The concrete types are defined in Core and all REL_EXP is defined in PK-Sim. I just made it this way to prevent moving these constant string into Core.

e.g. ```AllGlobalRelExpParameters```